app/experimenter/nimbus-ui/src/index.tsx

```diff
@@ -28,7 +28,7 @@ try {
     <React.StrictMode>
       <AppErrorBoundary>
         <ApolloProvider {...{ client }}>
-          <App />
+          <App basepath="/experiments/nimbus" />
```

Comment:
Ahh I was going to say this should be `/nimbus` (URL confirmed with Kate a few weeks ago) but it looks like that's how we're setup to serve this now. Since the links are relative `./new` if we do change from `experiments/nimbus` to `/nimbus` in a follow up hopefully it won't touch many files.